requirements.txt

```
@@ -1,7 +1,7 @@
 jupyter
 jupyter_client < 8.0.0
 tabulate
-shinylive==0.5.0
+shinylive @ git+https://github.com/posit-dev/py-shinylive@main
```

Don't merge until new release happens

```suggestion
shinylive==0.5.1
```

---

The new version of shinylive is on PyPI so I've made this change.